internal/acceptance/openstack/networking/v2/extensions/taas/tapmirrors_test.go

```diff
@@ -0,0 +1,39 @@
+//go:build acceptance || networking || taas
+
+package taas
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/v2/internal/acceptance/clients"
+	networking "github.com/gophercloud/gophercloud/v2/internal/acceptance/openstack/networking/v2"
+	"github.com/gophercloud/gophercloud/v2/internal/acceptance/tools"
+	th "github.com/gophercloud/gophercloud/v2/testhelper"
+)
+
+func TestTapMirrorCRUD(t *testing.T) {
```

Comment:
> CRUD

I don't see an Update func in this PR. Can you add it?

Reply:
This PR is only for Create, as the contribution guidelines said to separate the PRs when adding a suite of API calls. I see now that CRUD test name is misleading in this PR, would you rather I rename it now and then change it back in following PRs?

Reply:
Alright, I'll add everything to this PR. For reference, this is the contribution guideline I've mentioned https://github.com/gophercloud/gophercloud/blob/main/docs/contributor-tutorial/step-05-pull-requests.md#adding-a-suite-of-api-calls

Reply:
@kayrus  I have added the rest of the functions